packages/common/src/restHelpers.ts

```diff
@@ -127,13 +127,29 @@ export function convertExtent(
                       : undefined;
                   if (projectGeom && projectGeom.rings) {
                     const ring: any = projectGeom.rings[0];
-                    resolve({
-                      xmin: ring[0][0],
-                      ymin: ring[0][1],
-                      xmax: ring[2][0],
-                      ymax: ring[2][1],
-                      spatialReference: outSR
-                    });
+                    resolve(
+                      ring.reduce((accumulator: any, current: any) => {
```

Comment:
This seems to be more elaborate than we need; the extents to be projected are submitted clockwise starting in the southwestern corner, so the results should be the same. Is there a projection that we have to support for which this doesn't hold true?

Reply:
I could not answer that. Just so many pulls for one issue. I felt bad that it had come back to my queue yet again and went this way as I thought it was safer and regardless of the order they came back it should produce the expected values.

Reply:
Part of what threw me off in the first place was they are not returned in the order they are submitted so it made me question what I could rely on

Submitted as:
[[xmin, ymin],[xmin, ymax],[xmax, ymax],[xmax, ymin],[xmin, ymin]]

But returned as:
[[xmax, ymin],[xmin, ymin],[xmin, ymax],[xmax, ymax],[xmax, ymin]]

I'll gladly change back if you think that's best.